src/main/kotlin/com/criptext/mail/scenes/emaildetail/ui/holders/FullEmailHolder.kt

```diff
@@ -382,6 +385,22 @@ class FullEmailHolder(view: View) : ParentEmailHolder(view) {
         attachmentsRecyclerView.adapter?.notifyItemChanged(attachmentPosition)
     }
 
+    fun showStartGuideMenu(){
```

Comment:
If you need the activity here, you should use the IHostActivity and immplement the functions based on what you need, I think you could use an enum or class to let know the base activity which quick guide you need to show, and send the view as well for it to work. Then the BaseActivity handles this.

Reply:
We agreed with @saulmestanza that since there is no longer need for various different functions since the quick guide lib only needs the view to work, it can be abstracted and use only 1 functions with the View needed as a parameter and no need for the enum or sealed class.